### PR TITLE
Weight ticket checkboxes

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Add "missing weight ticket" checkboxes and warnings

## Setup
`make client_run`
`make server_run`

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/164718895) for this change

## Screenshots

image

### CODE REVIEW

File: cypress/integration/mymove/ppm.js [hunk 1]
```diff
@@ -100,50 +100,50 @@ describe('completing the ppm flow', function() {
...
-    cy.get('input[id="agree-checkbox"]').check({ force: true });
-    cy
-      .get('button')
-      .contains('Submit Payment')
-      .click();
-    cy.location().should(loc => {
-      expect(loc.pathname).to.match(/^\/$/);
-    });
-  });
+  // it('allows a SM to request payment', function() {
```
Comment (mkrump) [inline cypress/integration/mymove/ppm.js]: Isn't this the current production flow? Wasn't sure if this should be commented out or was just for development.

File: src/scenes/Moves/Ppm/WeightTicket.jsx [hunk 2]
```diff
@@ -68,11 +79,23 @@ class WeightTicket extends Component {
                   />{' '}
                   lbs
                 </div>
-                <div className="usa-width-two-third uploader-wrapper">
+                <div className="usa-width-two-thirds uploader-wrapper">
                   <Uploader options={{ labelIdle: uploadEmptyTicketLabel }} />
+                  <Checkbox
```
Comment (kahlouie) [inline src/scenes/Moves/Ppm/WeightTicket.jsx]: Got a warning when I clicked the checkbox before filling in any data other than vehicle type. See below.

```
index.js:1437 Warning: Failed prop type: The prop `heading` is marked as required in `Alert`, but its value is `undefined`.
    in Alert (at WeightTicket.jsx:92)
    in WeightTicket (created by Form(WeightTicket))
    in Form(WeightTicket) (created by Connect(Form(WeightTicket)))
```

File: cypress/integration/mymove/ppm.js (hunk 1 above)
Comment (Ronolibert) [reply cypress/integration/mymove/ppm.js]: Good catch, forgot to uncomment it after I was done writing the tests

File: src/scenes/Moves/Ppm/WeightTicket.jsx (hunk 2 above)
Comment (Ronolibert) [reply src/scenes/Moves/Ppm/WeightTicket.jsx]: Changed this to the Alert component require either `heading` or `children`

**Pull request merged**